source/_posts/similarity.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Similarity and Clustering
```

Comment:
Title   tag character should be 50-60 character, it is right now 38, Please update it